picky-signtool/src/utils.rs

```diff
@@ -0,0 +1,4 @@
+pub fn str_to_utf16_bytes(s: &str, buff: &mut Vec<u8>) {
+    s.encode_utf16()
+        .for_each(|word| buff.extend_from_slice(&word.to_le_bytes()));
+}
```

Comment:
This could be written as:
```suggestion
pub fn str_to_utf16_bytes(s: &str, buff: &mut Vec<u8>) {
    buff.extend(s.encode_utf16().map(|word| word.to_le_bytes()).flatten())
}
```